pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.black]
+line-length = 88 # Black's default value
+target-version = ["py36"]
+
+[isort]
+extend_skip = ["*migrations*", "metarecord/binding"]
+profile = "black"
+atomic = true
+order_by_type = false
+py_version = 36
+
+[tool.pytest]

```

Comment:
pytest recommends/tells to use `[tool.pytest.ini_options]` in https://docs.pytest.org/en/7.2.x/reference/customize.html#pyproject-toml (Kind of wondering why I didn't do it for linkedevents :shrug:)

Reply:
Well spotted!